selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -464,6 +470,28 @@ private void captureScreenshotsFromOpenedWindows(
             });
   }
 
+  private void storeWebDriverLogs(ITestResult result) {
+    Set<SeleniumWebDriver> webDrivers = new HashSet<>();
+    Object testInstance = result.getInstance();
+    collectInjectedWebDrivers(testInstance, webDrivers);
+    webDrivers.forEach(webDriver -> getWebDriverLog(result, webDriver));
+  }
+
+  private void getWebDriverLog(ITestResult result, SeleniumWebDriver webDriver) {
```

Comment:
`get` usually means returning something.
This method returns nothing. It store logs of webdriver into into the filesystem. So, method name should be rephrased appropriately, for example: `storeWebDriverLogs`